code/js/rcalc-log.js

```diff
@@ -0,0 +1,46 @@
+var lineCounter = NUMBER_UTILITY_CONST.NUMBER_ZERO;
+
+function setLogDisplayLine() {
+  lineCounter++;
+  var parentDiv = document.getElementById(HTML_ID_CONST.ID_LOG_DISPLAY);
+  var paragraph = document.createElement(STRING_ATTRIBUTE_ELEMENT_CONST.ELEMENT_P);
+  var text;
+  var nextToFirstChild;
+  paragraph.style.backgroundColor = HTML_CSS_CONST.COLOR_LIGHT_GREY;
+  paragraph.setAttribute(STRING_ATTRIBUTE_ELEMENT_CONST.ATTRIBUTE_CLASS, HTML_CLASS_CONST.RCALC_LOG_DISPLAY_LINE);
+  text = document.createTextNode(leftSideOfEqualSignGlobal + STRING_UTILITY_CONST.EQUALS_SIGN + resultGlobal);
+  paragraph.appendChild(text);
+  parentDiv.insertBefore(paragraph, parentDiv.firstChild);
+  nextToFirstChild = parentDiv.firstChild.nextSibling;
```

Comment:
nextToFirstChild = parentDiv.firstChild.nextSibling;
  parentDiv.removeChild(nextToFirstChild);
... do it by one line without var nextToFirstChild

Reply:
only one line like this parentDiv.removeChild(parentDiv.firstChild.nextSibling);
var nextToFirstChild removed... no needed